ODLA/platforms/dnnl/odla_dnnl.h

```diff
@@ -280,4 +280,13 @@ static inline odla_value CreateValue(const dnnl::memory& mem,
   return ret;
 }
 
+static dnnl::memory broadcast_mem(const dnnl::memory orig_mem,
```

Comment:
static declaration won't work for odla_dnnl_binary.cc

Reply:
you can either move the definition here (like line 273) or follow the example in line 36.